tests/director/tests/runtime_api_test.go

```diff
@@ -441,8 +441,8 @@ func TestRegisterUpdateRuntimeWithoutLabels(t *testing.T) {
 
 	tenantId := tenant.TestTenants.GetDefaultTenantID()
 
-	fixtures.CreateFormationWithinTenant(t, ctx, certSecuredGraphQLClient, tenantId, testScenario)
 	defer fixtures.DeleteFormationWithinTenant(t, ctx, certSecuredGraphQLClient, tenantId, testScenario)
```

Comment:
i am not sure how this improves the clean up

Reply:
The 'defer' needs to be before the CreateFormationWithinTenant because this method has 2 assertions and if one of them fails then the defer will not be executed.